### PR TITLE
Stop triggering `select-all-notifications-shortcut` when typing

### PR DESCRIPTION
**Pull request by cheap-glitch**
Fixes the feature being triggered when typing in the search field :man_facepalming: 

## Test URLs

https://github.com/notifications

## Screenshot

https://user-images.githubusercontent.com/46634000/162978463-6add5495-704a-4683-bb7f-c15c0fae1726.mp4

### CODE REVIEW

File: source/features/select-all-notifications-shortcut.tsx [hunk 1]
```diff
@@ -27,7 +28,7 @@ void features.add(import.meta.url, {
 		pageDetect.isNotifications,
 	],
 	exclude: [
-		() => select.exists('img[src$="notifications/inbox-zero.svg"]'), // Don't run on empty inbox page
+		pageDetect.isBlank, // Empty notification list
```
Comment (cheap-glitch) [inline source/features/select-all-notifications-shortcut.tsx:31]: This excludes every empty notification tabs ("Inbox", "Saved", etc.)

**Review by fregante — CHANGES_REQUESTED** [review]
This is why I'd rather have a helper that generates `data-hotkey` element.

If this is a bug, let's hotfix the feature for now and just use the code I suggested

File: source/github-helpers/register-hotkey.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'dom-chef';
+
+export default function registerHotkey(hotkey: string, handler: VoidFunction): Deinit {
+	const hotkeyButton = <button hidden type="button" data-hotkey={hotkey} onClick={handler}/>;
+	document.body.append(hotkeyButton);
+
+	return () => {
+		if (hotkeyButton.isConnected) {
+			hotkeyButton.remove();
+		}
```
Comment (fregante) [inline source/github-helpers/register-hotkey.tsx:10]: This won't throw
```suggestion
		hotkeyButton.remove();
```

**Review by fregante — APPROVED** [review]
✨

**Pull request merged**